Makefile

```diff
@@ -5,7 +5,7 @@ update:
 	./composer.phar update
 
 test: vendor/bin/phpunit build
-	./vendor/bin/phpunit --strict
+	./vendor/bin/phpunit
```

Comment:
The command line flag --strict is deprecated.  This is due to an upgrade of phpunit; as is the change from assertTag to simple regex.